server/websocket_test.go

```diff
@@ -601,6 +601,9 @@ func TestHandleJoin(t *testing.T) {
 	mockAPI.On("LogInfo", mock.Anything, mock.Anything, mock.Anything,
 		mock.Anything, mock.Anything, mock.Anything, mock.Anything,
 		mock.Anything, mock.Anything, mock.Anything, mock.Anything)
+	mockAPI.On("LogWarn", mock.Anything, mock.Anything, mock.Anything,
```

Comment:
This was purely bc of mac -- mac warns `failed to set udp send buffer` and `failed to set udp receive buffer`. I haven't looked into it seriously, it doesn't seem to be a widespread issue, and since we don't support it in production I haven't spent more time on it.